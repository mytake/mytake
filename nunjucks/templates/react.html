{% extends "base.html" %}

{% block endbody %}
  <!-- Dependencies -->
  <script src="https://unpkg.com/react@15/dist/react.js"></script>
  <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>

  <!-- Main -->
  <script src="./bundle-0.0.3.js"></script>
{% endblock %}
