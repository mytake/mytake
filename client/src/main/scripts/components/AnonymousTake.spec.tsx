import * as React from "react";
import * as renderer from "react-test-renderer";
import AnonymousTake from "./AnonymousTake";
import { anonymousPath } from "../utils/testUtils";

jest.mock("./BlockEditor", () => ({
  default: "BlockEditor"
}));

test("It renders", () => {
  const tree = renderer.create(<AnonymousTake path={anonymousPath} />).toJSON();
  expect(tree).toMatchSnapshot();
});
