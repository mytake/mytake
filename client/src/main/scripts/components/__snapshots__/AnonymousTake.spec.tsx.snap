// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders 1`] = `
<div
  className="anonymoustake"
>
  <div>
    <BlockEditor
      eventHandlers={
        Object {
          "onDocumentClick": [Function],
        }
      }
      takeDocument={
        Object {
          "blocks": Array [
            Object {
              "kind": "video",
              "range": Array [
                0.1,
                5,
              ],
              "videoId": "iEfwIxM7MmnOKb7zt4HqW8IxUWy6F7a236fSOQlUUWI=",
            },
          ],
          "title": "Introduction",
        }
      }
    />
    <p
      className="anonymoustake__text"
    >
      This take was made by someone without a MyTake.org account.
       
      <a
        className="anonymoustake__link"
        href="/login"
      >
        Create an account
      </a>
       
      to get access to powerful authoring tools as well as the ability to track the number of likes and views for your own takes.
    </p>
  </div>
</div>
`;
